l10n_it_account_stamp/views/invoice_view.xml

```diff
@@ -20,7 +20,8 @@
                 <field name="invoice_line_ids" position="after">
                     <form>
                         <group class="oe_left">
-                            <field name="tax_stamp" string="Tax stamp"/>
+                            <field name="auto_compute_stamp" invisible="1"/>
+                            <field name="tax_stamp" string="Tax stamp" attrs="{'readonly': [('auto_compute_stamp', '=', True)]}"/>
```

Comment:
@primes2h readonly fields are not sent to server:

![peek 2019-01-23 18-02](https://user-images.githubusercontent.com/1033131/51623822-d5148700-1f39-11e9-82f3-dbd3a192341b.gif)

Reply:
This is fixed.
Now checkbox is invisible in automatic mode. Stamp application is visible using an image when correct amount is reached (in automatic mode) or checkbox is selected  (manual mode).
I added other fixes too.